dev/tasks/vcpkg-tests/cpp-build-vcpkg.bat

```diff
@@ -0,0 +1,69 @@
+@rem Licensed to the Apache Software Foundation (ASF) under one
+@rem or more contributor license agreements.  See the NOTICE file
+@rem distributed with this work for additional information
+@rem regarding copyright ownership.  The ASF licenses this file
+@rem to you under the Apache License, Version 2.0 (the
+@rem "License"); you may not use this file except in compliance
+@rem with the License.  You may obtain a copy of the License at
+@rem
+@rem   http://www.apache.org/licenses/LICENSE-2.0
+@rem
+@rem Unless required by applicable law or agreed to in writing,
+@rem software distributed under the License is distributed on an
+@rem "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+@rem KIND, either express or implied.  See the License for the
+@rem specific language governing permissions and limitations
+@rem under the License.
+
+@rem Run VsDevCmd.bat to set Visual Studio environment variables for building
+@rem on the command line. This is the path for Visual Studio Enterprise 2019
+call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat" -arch=amd64
```

Comment:
I'm curious whether this is necessary. Isn't CMake able to detect the VS install? Or is this for vcpkg?

Reply:
The CMake that's being used here is the one distributed with Visual Studio. This command adds that CMake to the path and sets some environment variables to make it work correctly. If I omit this, then the `cmake` command below gives `'cmake' is not recognized as an internal or external command,
operable program or batch file.`

Reply:
Ow, I forgot that the vcpkg model is to build everything from source. So this is gonna rebuilt OpenSSL, Boost, gRPC... everytime.

I understand why you didn't enable Gandiva!